src/index.html

```diff
@@ -11,7 +11,170 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset id="first">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            autocomplete="off"
+            name="surname"
+            required
+          >
+        </div>
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+            required
+          >
+        </div>
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            min="1"
+            max="100"
+            id="howoldareyou"
+            name="howoldareyou"
+            value="12"
+            required
+          >
+        </div>
+        <div class="form-field">
+          <label for="date">Full date of birth:</label>
+          <input
+            type="date"
+            id="date"
+            name="date"
+            required
+          >
+        </div>
+        <div class="form-field">
+          <label for="terms-and-conditions">
+            I accept the term of the agreement
+          </label>
+          <input
+            type="checkbox"
+            id="terms-and-conditions"
+            name="terms"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+        <div class="form-field">
+          <label for="name">Password:</label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field"> Do you love cats?
+          <input
+            type="radio"
+            id="yes"
+            name="cat"
+          >
+          <label for="yes">Yes</label>
+          <input
+            type="radio"
+            id="no"
+            name="cat"
+          >
+          <label for="no">No</label>
+        </div>
+        <div class="form-field">
+          <label for="color">What is your favorite color?
```

Comment:
You should be consistent about your code style. If you move closing tab to the new line - text should be on a separate line too
```suggestion
          <label for="color">
            What is your favorite color?
```